ood-portal-generator/spec/fixtures/sum.default

```diff
@@ -1 +1 @@
-4fe33e88acaf968dd2f601215d9a5206fac16004a0d47c9d95be91cf2f7b8edf /opt/rh/httpd24/root/etc/httpd/conf.d/ood-portal.conf
+7290aeee956bf3ab1aec55a3c2b53669b5abd6befca79667e87fe7901c558d94
```

Comment:
This change looks incorrect.  The file should contain the filename which is what happens when really deployed.
```suggestion
7290aeee956bf3ab1aec55a3c2b53669b5abd6befca79667e87fe7901c558d94 /opt/rh/httpd24/root/etc/httpd/conf.d/ood-portal.conf
```

Reply:
Thanks, I just redirected output without thinking.